keyboards/dtd/ldmlKeyboard3.xsd

```diff
@@ -50,16 +50,11 @@ Note: DTD @-annotations are not currently converted to .xsd. For full CLDR file
   <xs:element name="import">
     <xs:complexType>
       <xs:attribute name="path" use="required"/>
-      <xs:attribute name="base">
-        <xs:simpleType>
-          <xs:restriction base="xs:token">
-            <xs:enumeration value="cldr"/>
-          </xs:restriction>
-        </xs:simpleType>
-      </xs:attribute>
+      <xs:attribute name="base" type="xs:NMTOKEN"/>
```

Comment:
The spec says:

> The base may be omitted (indicating a local import) or have the value "cldr".

Whereas this change is allowing any value in `base`, isn't it? 

As far as I can tell, the previous .xsd was actually correct? It doesn't have `use="required"` but it does restrict the allowable values to 'cldr'.

I think the DTD change looks okay though.

Reply:
I don't mind allowing different bases, it is up to implementations which sources they want to support.